onnx/defs/math/defs.cc

```diff
@@ -1070,7 +1008,21 @@ ONNX_OPERATOR_SET_SCHEMA(
 
               schema.BuildFunction(functionProto);
               return true;
-            }));
+            })
+        .SetContextDependentFunctionBodyBuilder(
+            [](const FunctionBodyBuildContext& ctx, const OpSchema& schema, FunctionProto& functionProto) -> bool {
+              int64_t axis = ctx.getAttribute("axis") != nullptr ? ctx.getAttribute("axis")->i() : -1;
+              FunctionBuilder builder(functionProto);
+              builder.Const1D("axes", axis).Add("X_ReduceMax = ReduceMax <keepdims = 1> (input, axes)").Add(R"(
```

Comment:
Nit: Is this the auto-formatter doing this? It's easier to read if we have one (ONNX op) per line.

Reply:
I agree. Formatting rules enforced is less then great. Many times I find it making code ugly. It refuses my manual correction.